SSDynamicText/SSDynamicButton.m

```diff
@@ -63,15 +83,31 @@ - (void)dealloc {
     [self ss_stopObservingTextSizeChanges];
 }
 
+- (void)changeFontWithDelta:(NSInteger)newDelta {
+    CGFloat preferredSize = [self.defaultFontDescriptor.fontAttributes[UIFontDescriptorSizeAttribute] floatValue];
+    preferredSize += newDelta;
+
+    self.titleLabel.font = [UIFont fontWithDescriptor:self.defaultFontDescriptor
+                                                 size:preferredSize];
+}
+
+- (void)changeAttributedTitle:(NSAttributedString *)attributedTitle forState:(UIControlState)state withFontSizeDelta:(NSInteger)newDelta {
+    [super setAttributedTitle:[attributedTitle ss_attributedStringWithAdjustedFontSizeWithDelta:newDelta] forState:state];
```

Comment:
Well, each class inherits from UIView, and UIView extension is a bad idea to reduce this code repetition. I think it should be left as is.

As for super, in [self setAttributedTitle] we change baseAttributedText, under which is based newDelta count. If we would call it when size changes - we would change baseAttributedText so delta would be always 0. I call here super to set new AttributedText without changing baseAttributedText.